bin/end_to_end.sh

```diff
@@ -57,6 +57,9 @@ export SIGNON_USERNAME="$(get_param /${environment}/smoketests/signon/username)"
 export SIGNON_OTP="$(get_param /${environment}/smoketests/signon/secret)"
 export SIGNON_PASSWORD="$(get_param /${environment}/smoketests/signon/password)"
 export SETTINGS__GOVUK_NOTIFY__API_KEY="$(get_param /${environment}/smoketests/notify/api-key)"
+export AUTH0_EMAIL_USERNAME="$(get_param /${environment}/smoketests/auth0/email-username)"
+export AUTH0_GMAIL_ADDRESS="$(get_param /${environment}/smoketests/auth0/gmail-address)"
```

Comment:
I don't understand what the difference is between `EMAIL_USERNAME` and `GMAIL_ADDRESS` is... is that related to google groups, or is it something else?

Reply:
yes - email username is the google group. The gmail address is a gmail account which is subscribed to the google group. The gmail address doesn't have a .gov.uk address so we use a google group which does.

When logging in to auth0 we use the google group email address.

The gmail address is what we actually check in the tests - the group receives the email and forwards it to the gmail address.

Reply:
Ah okay, thanks for explaining. Maybe a comment or a more explicit name would be good, just so it's clearer in the code?

Reply:
I've added some comment to the values in the complete_spec rather than here